pavelib/prereqs.py

```diff
@@ -315,3 +315,13 @@ def log_installed_python_prereqs():
     """  Logs output of pip freeze for debugging. """
     sh("pip freeze > {}".format(Env.GEN_LOG_DIR + "/pip_freeze.log"))
     return
+
+
+def print_devstack_warning():  # pylint: disable=missing-docstring
```

Comment:
Is this meant to be called from somewhere?

Reply:
I tried to slip in a rebased fix before you reviewed, but you caught this to early.  Unfortunately, my fix got killed in a rebase and I had to rewrite this.